CRM/Report/Form/Contact/Relationship.php

```diff
@@ -528,13 +561,12 @@ public function where() {
         }
       }
     }
-
-    if (empty($whereClauses)) {
-      $this->_where = 'WHERE ( 1 ) ';
-      $this->_having = '';
+    $this->_where = "WHERE ( {$this->_aliases['civicrm_contact']}.is_deleted = 0 AND {$this->_aliases['civicrm_contact_b']}.is_deleted = 0 ) ";
```

Comment:
@Edzelopez @colemanw what do you think of this. I took a look at a few other reports & they weren't filtering out deleted contacts. I kind of feel like if we want them filtered out the generated ACL clause should be doing that for all reports?

Overall PR looks good, just trying to think it through WRT report ACL work that has been happening